Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -4672,10 +4672,8 @@ RefPtr<CSSValue> consumeImage(CSSParserTokenRange& range, const CSSParserContext
             return consumeGeneratedImage(range, context);
 
         if (allowedImageTypes.contains(AllowedImageType::ImageSet)) {
-            if (functionId == CSSValueImageSet)
+            if (functionId == CSSValueImageSet || functionId == CSSValueWebkitImageSet)
                 return consumeImageSet(range, context, (allowedImageTypes | AllowedImageType::RawStringAsURL) - AllowedImageType::ImageSet);
```

Comment:
I wonder if consumeImageSet can be simplified.

Reply:
It will definitely get some love when [image-set does not support type](https://bugs.webkit.org/show_bug.cgi?id=225185) gets worked on.